src/utils/firebaseOperations.js

```diff
@@ -0,0 +1,39 @@
+// Import the necessary Firestore functions
+import { doc, updateDoc, increment } from 'firebase/firestore';
```

Comment:
we will use react-firebase-query function, not functions from firebase

Reply:
https://react-query-firebase.invertase.dev/firestore/data-mutation

Reply:
also, you can create a 'hooks' or 'utils' folder under the 'Dictionary' page folder and put these functions

Reply:
I think you can use Anthony's getEntriesMutation.js
 https://github.com/rainawan/BruinDictionary/pull/24/files